src/UIKit/UIAccessibility.cs

```diff
@@ -339,6 +346,20 @@ static unsafe void TrampolineRequestGuidedAccessSession (IntPtr block, bool enab
 			}
 		}
 
+		[iOS (13,0), TV (13,0)]
+		[DllImport (Constants.UIKitLibrary)]
```

Comment:
Is Constants.UIKitLibrary a third party dll?

Reply:
Nope, `UIKitLibrary` points to the system's path where this library is located, the value changes between simulator and devices.